build-system/tasks/performance/config.json

```diff
@@ -1,5 +1,6 @@
 {
   "headless": true,
   "runs": 100,
+  "timeout": 2000,
```

Comment:
If I can ask, this will apply 2000 timeout to all tests with requestHandler. How would you customize it for different use case?

Reply:
Hmm not sure. Can you give an example where another timeout is needed? Do you think it will happen often?

Reply:
Discussed offline, agreed to try out having a config section of each request handler to specify their  metrics, such as URLs and timeout.